can't install on macOS 15

```
edwineefting@edwins-MacBook-Pro ~/tmp> npm install freetype2
npm error code 1
npm error path /Users/edwineefting/tmp/node_modules/freetype2
npm error command failed
npm error command sh -c node-gyp-build
npm error CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftsystem.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftinit.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftdebug.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftbase.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftbbox.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftglyph.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftbdf.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftbitmap.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftcid.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftfstype.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftgasp.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftgxval.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftlcdfil.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftmm.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftotval.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftpatent.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftpfr.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftstroke.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftsynth.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/fttype1.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/base/ftwinfnt.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/bdf/bdf.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/cff/cff.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/cid/type1cid.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/pcf/pcf.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/pfr/pfr.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/sfnt/sfnt.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/truetype/truetype.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/type1/type1.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/type42/type42.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/winfonts/winfnt.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/raster/raster.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/smooth/smooth.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/autofit/autofit.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/cache/ftcache.o
npm error   CC(target) Release/obj.target/libfreetype/vendor/freetype/src/gzip/ftgzip.o
npm error gyp info it worked if it ends with ok
npm error gyp info using node-gyp@11.0.0
npm error gyp info using node@23.7.0 | darwin | arm64
npm error gyp info find Python using Python version 3.9.6 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
npm error gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
npm error gyp info spawn args [
npm error gyp info spawn args '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm error gyp info spawn args 'binding.gyp',
npm error gyp info spawn args '-f',
npm error gyp info spawn args 'make',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/Users/edwineefting/tmp/node_modules/freetype2/build/config.gypi',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/Users/edwineefting/Library/Caches/node-gyp/23.7.0/include/node/common.gypi',
npm error gyp info spawn args '-Dlibrary=shared_library',
npm error gyp info spawn args '-Dvisibility=default',
npm error gyp info spawn args '-Dnode_root_dir=/Users/edwineefting/Library/Caches/node-gyp/23.7.0',
npm error gyp info spawn args '-Dnode_gyp_dir=/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp',
npm error gyp info spawn args '-Dnode_lib_file=/Users/edwineefting/Library/Caches/node-gyp/23.7.0/<(target_arch)/node.lib',
npm error gyp info spawn args '-Dmodule_root_dir=/Users/edwineefting/tmp/node_modules/freetype2',
npm error gyp info spawn args '-Dnode_engine=v8',
npm error gyp info spawn args '--depth=.',
npm error gyp info spawn args '--no-parallel',
npm error gyp info spawn args '--generator-output',
npm error gyp info spawn args 'build',
npm error gyp info spawn args '-Goutput_dir=.'
npm error gyp info spawn args ]
npm error gyp info spawn make
npm error gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm error In file included from ../vendor/freetype/src/gzip/ftgzip.c:72:
npm error In file included from ../vendor/freetype/src/gzip/zlib.h:34:
npm error ../vendor/freetype/src/gzip/ftzconf.h:228:12: error: unknown type name 'Byte'
npm error   228 |    typedef Byte  FAR Bytef;
npm error       |            ^
npm error In file included from ../vendor/freetype/src/gzip/ftgzip.c:90:
npm error ../vendor/freetype/src/gzip/inftrees.h:22:7: error: unknown type name 'Byte'
npm error    22 |       Byte Exop;        /* number of extra bits or operation */
npm error       |       ^
npm error ../vendor/freetype/src/gzip/inftrees.h:23:7: error: unknown type name 'Byte'
npm error    23 |       Byte Bits;        /* number of bits in this code or subcode */
npm error       |       ^
npm error In file included from ../vendor/freetype/src/gzip/ftgzip.c:98:
npm error ../vendor/freetype/src/gzip/inftrees.c:247:21: error: use of undeclared identifier 'Byte'
npm error   247 |           r.bits = (Byte)l;     /* bits to dump before this table */
npm error       |                     ^
npm error ../vendor/freetype/src/gzip/inftrees.c:248:21: error: use of undeclared identifier 'Byte'
npm error   248 |           r.exop = (Byte)j;     /* bits in this table */
npm error       |                     ^
npm error ../vendor/freetype/src/gzip/inftrees.c:258:17: error: use of undeclared identifier 'Byte'
npm error   258 |       r.bits = (Byte)(k - w);
npm error       |                 ^
npm error ../vendor/freetype/src/gzip/inftrees.c:263:19: error: use of undeclared identifier 'Byte'
npm error   263 |         r.exop = (Byte)(*p < 256 ? 0 : 32 + 64);     /* 256 is end-of-block */
npm error       |                   ^
npm error ../vendor/freetype/src/gzip/inftrees.c:268:19: error: use of undeclared identifier 'Byte'
npm error   268 |         r.exop = (Byte)(e[*p - s] + 16 + 64);/* non-simple--look up in lists */
npm error       |                   ^
npm error In file included from ../vendor/freetype/src/gzip/ftgzip.c:100:
npm error ../vendor/freetype/src/gzip/infcodes.c:50:3: error: unknown type name 'Byte'
npm error    50 |   Byte lbits;           /* ltree bits decoded per branch */
npm error       |   ^
npm error ../vendor/freetype/src/gzip/infcodes.c:51:3: error: unknown type name 'Byte'
npm error    51 |   Byte dbits;           /* dtree bits decoder per branch */
npm error       |   ^
npm error ../vendor/freetype/src/gzip/infcodes.c:70:17: error: use of undeclared identifier 'Byte'
npm error    70 |     c->lbits = (Byte)bl;
npm error       |                 ^
npm error ../vendor/freetype/src/gzip/infcodes.c:71:17: error: use of undeclared identifier 'Byte'
npm error    71 |     c->dbits = (Byte)bd;
npm error       |                 ^
npm error ../vendor/freetype/src/gzip/infcodes.c:207:9: error: use of undeclared identifier 'Byte'
npm error   207 |         OUTBYTE(*f++)
npm error       |         ^
npm error ../vendor/freetype/src/gzip/infutil.h:83:27: note: expanded from macro 'OUTBYTE'
npm error    83 | #define OUTBYTE(a) {*q++=(Byte)(a);m--;}
npm error       |                           ^
npm error In file included from ../vendor/freetype/src/gzip/ftgzip.c:100:
npm error ../vendor/freetype/src/gzip/infcodes.c:216:7: error: use of undeclared identifier 'Byte'
npm error   216 |       OUTBYTE(c->sub.lit)
npm error       |       ^
npm error ../vendor/freetype/src/gzip/infutil.h:83:27: note: expanded from macro 'OUTBYTE'
npm error    83 | #define OUTBYTE(a) {*q++=(Byte)(a);m--;}
npm error       |                           ^
npm error ../vendor/freetype/src/gzip/ftgzip.c:312:23: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte[4096]' (aka 'unsigned char[4096]') [-Wincompatible-pointer-types]
npm error   312 |     zstream->next_in  = zip->buffer;
npm error       |                       ^ ~~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:361:26: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte[4096]' (aka 'unsigned char[4096]') [-Wincompatible-pointer-types]
npm error   361 |       zstream->next_in   = zip->input;
npm error       |                          ^ ~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:363:26: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte[4096]' (aka 'unsigned char[4096]') [-Wincompatible-pointer-types]
npm error   363 |       zstream->next_out  = zip->buffer;
npm error       |                          ^ ~~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:408:23: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte[4096]' (aka 'unsigned char[4096]') [-Wincompatible-pointer-types]
npm error   408 |     zstream->next_in  = zip->input;
npm error       |                       ^ ~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:423:24: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte *' (aka 'unsigned char *') [-Wincompatible-pointer-types]
npm error   423 |     zstream->next_out  = zip->cursor;
npm error       |                        ^ ~~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:442:20: warning: incompatible pointer types assigning to 'FT_Byte *' (aka 'unsigned char *') from 'Bytef *' (aka 'int *') [-Wincompatible-pointer-types]
npm error   442 |         zip->limit = zstream->next_out;
npm error       |                    ^ ~~~~~~~~~~~~~~~~~
npm error ../vendor/freetype/src/gzip/ftgzip.c:741:22: warning: incompatible pointer types assigning to 'Bytef *' (aka 'int *') from 'FT_Byte *' (aka 'unsigned char *') [-Wincompatible-pointer-types]
npm error   741 |     stream.next_out  = output;
npm error       |                      ^ ~~~~~~
npm error 7 warnings and 14 errors generated.
npm error make: *** [Release/obj.target/libfreetype/vendor/freetype/src/gzip/ftgzip.o] Error 1
npm error gyp ERR! build error
npm error gyp ERR! stack Error: `make` failed with exit code: 2
npm error gyp ERR! stack at ChildProcess.<anonymous> (/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:216:23)
npm error gyp ERR! System Darwin 24.3.0
npm error gyp ERR! command "/opt/homebrew/Cellar/node/23.7.0/bin/node" "/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm error gyp ERR! cwd /Users/edwineefting/tmp/node_modules/freetype2
npm error gyp ERR! node -v v23.7.0
npm error gyp ERR! node-gyp -v v11.0.0
npm error gyp ERR! not ok
npm error A complete log of this run can be found in: /Users/edwineefting/.npm/_logs/2025-02-27T19_40_50_907Z-debug-0.log
edwineefting@edwins-MacBook-Pro ~/tmp [1]> node --version
v23.7.0
edwineefting@edwins-MacBook-Pro ~/tmp> uname -a
Darwin edwins-MacBook-Pro.local 24.3.0 Darwin Kernel Version 24.3.0: Thu Jan  2 20:22:58 PST 2025; root:xnu-11215.81.4~3/RELEASE_ARM64_T8132 arm64
edwineefting@edwins-MacBook-Pro ~/tmp>

edwineefting@edwins-MacBook-Pro ~/tmp> sw_vers
ProductName:		macOS
ProductVersion:		15.3.1
BuildVersion:		24D70
```

--- a/.clangd
+++ b/.clangd
@@ -1,0 +1,2 @@
+CompileFlags:
+  CompilationDatabase: build/Debug

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
   "scripts": {
     "install": "pkg-prebuilds-verify ./binding-options.js || node-gyp rebuild",
     "build": "node-gyp build",
+    "build.dev": "node-gyp configure --debug && node-gyp configure --debug -- -f compile_commands_json && node-gyp build --debug",
     "rebuild": "node-gyp clean configure build",
     "test": "jest"
   },

